src/main/java/com/google/summit/ast/modifier/Modifiable.kt

```diff
@@ -0,0 +1,18 @@
+package com.google.summit.ast.modifier
+
+/** Indicates that a [Node][com.google.summit.ast.Node] can be modified by [Modifier]s. */
+interface Modifiable {
```

Comment:
Sorry if this is a bit late to point this out, but the more I think about this name the less I like it. It seems to indicate something like "Mutable" vs "Immutable".

I suggest this interface be called "HasModifiers" instead. "MayHaveModifiers" is more pedantically correct, though a bit clunky.

Reply:
I take the blame for this name. `HasModifiers` LGTM